app/src/main/java/com/gathering/android/profile/updateinfo/UpdateUserInfoFragment.kt

```diff
@@ -0,0 +1,128 @@
+package com.gathering.android.profile.updateinfo
+
+import android.graphics.drawable.BitmapDrawable
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.widget.doOnTextChanged
+import androidx.fragment.app.DialogFragment
+import androidx.navigation.fragment.findNavController
+import com.gathering.android.R
+import com.gathering.android.auth.model.User
+import com.gathering.android.common.ImageLoader
+import com.gathering.android.common.getNavigationResultLiveData
+import com.gathering.android.common.setNavigationResult
+import com.gathering.android.databinding.BottomSheetUpdateUserInfoBinding
+import com.gathering.android.event.KEY_ARGUMENT_SELECTED_IMAGE
+import com.gathering.android.event.KEY_ARGUMENT_UPDATE_USER_INFO_DATA
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class UpdateUserInfoFragment : BottomSheetDialogFragment() {
+
+    lateinit var binding: BottomSheetUpdateUserInfoBinding
+
+    @Inject
+    lateinit var viewModel: UpdateUserInfoViewModel
+
+    @Inject
+    lateinit var imageLoader: ImageLoader
+
+    private var photoUrl: String = ""
```

Comment:
Same here as well.

Reply:
we have update_user_info layout instead of a button right now :)

Reply:
it's solved